notebooks/dataset_tutorial/README.md

```diff
@@ -1,21 +1,62 @@
+  - **images:** This directory contains NumPy arrays where each single `X.npy` contains the image that was shown to the mouse in trial `X`.
+  - **responses:** This directory contains NumPy arrays where each single `X.npy` contains the deconvolved calcium traces (i.e. responses) recorded from the mouse in trial `X` in response to the particular presented image.
+  - **behavior:** Behavioral variables include pupil dilation, the derivative of the pupil size, and running speed. The directory contain NumPy arrays (of size `1 x 3`) where each single `X.npy` contains the behavioral variables (in the same order that was mentioned earlier) for trial `X`.
+  - **pupil_center:** the eye position of the mouse, estimated as the center of the pupil. The directory contain NumPy arrays (of size `1 x 2`) for horizontal and vertical eye positions.
 - **meta:** includes meta data of the experiment
     - **neurons:** This directory contains neuron-specific information. Below are a list of important variables in this directory
         - `area.npy`: contains the area of each neuron
-        - `cell_motor_coordinates.npy`: contains the position (x, y, z) of each neuron in the cortex
+        - `cell_motor_coordinates.npy`: contains the position (x, y, z) of each neuron in the cortex, given in microns. **Note:** The
         - `layer.npy`: contains the cortex layer to which neuron belongs to
         - `unit_ids.npy`: contains a unique id for each neuron
     - **statistics:** This directory contains statistics (i.e. mean, median, etc.) of the experimental variables (i.e. behavior, images, pupil_center, and responses).
-    - **trials:** This directory contains trial-specific meta data. Below are a list of important variables in this directory:
-        - `frame_image_id.npy`: contains unique image id. If the image is presented multiple times (which is the case in the test set) the image id does not change.
-        - `tiers.npy`: contains labels that are used to split the data into *train*, *validation*, and *test* set.
-        - `trial_idx.npy`: contains unique (hashed) index for each trial. While the true trial index is available for the “pre-training” datasets, it is hidden (i.e. hashed) in the competition datasets. As opposed to the image ids, the trial index is different for different trials even if the image stays the same.
\ No newline at end of file
+      - **Note:** The statistics of the responses are or particular importance, because we provide the deconvolved calcium traces here in the responses.
+      
+        However, for the evaluation of submissions in the competition, we require the responses to be **standardized** (i.e. `r = r/(std_r)`).
```

Comment:
Could you make it clear that the STD is computed across all images in the training set?

Reply:
fixed